### DISCUSSION TITLE
cant run the cucumber cli

### DISCUSSION THREAD

**Discussion by invalid-code** [cucumber-rs/cucumber#277] [Q&A] [answered]
when I try to 
`cargo test -p cucumber --test example`
I get this error 
`error: package 'cucumber' cannot be tested because it requires dev-dependencies and is not a member of the workspace`
I installed cucumber as a dev dependency as stated in the documentation
`[dev-dependencies]
cucumber = "0.19.1"`

**Comment by ilslv** [comment, marked as answer]
@invalid-code try running `cargo test --test example`. Specifying concrete package with `-p`/`--package` is useful when you have multiple crates inside a single directory, like this repo has: `cucumber` and `cucumber-codegen`.

**Reply by invalid-code** [reply]
hm... it seems to work now.
I dont know why it didnt work before
even with `cargo test`
which is also working now
